pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -245,6 +245,19 @@ protected Policies getNamespacePolicies(String property, String cluster, String
         }
     }
 
+    protected LocalPolicies getNamespaceLocalPolicies(String property, String cluster, String namespace) {
```

Comment:
> Shouldn't we configuring the throttling policies on the global zk?

Actually, every cluster has different traffic for the namespace and we may want to control enabling or throttling-limit per cluster level. Therefore, kept it in local-policies and not in global-zk.